modular_skyrat/modules/customization/modules/clothing/shoes/shoes.dm

```diff
@@ -39,6 +39,19 @@
 	name = "high heels"
 	desc = "A fancy pair of high heels. Won't compensate for your below average height that much."
 	icon_state = "heels"
+	greyscale_config = /datum/greyscale_config/heels
+	greyscale_config_worn = /datum/greyscale_config/heels/worn
+	greyscale_config_worn_digi = /datum/greyscale_config/heels/worn/digi
+	greyscale_colors = "#FFFFFF"
+	flags_1 = IS_PLAYER_COLORABLE_1
+	body_parts_covered = TRUE
```

Comment:
Are you sure this is a TRUE/FALSE kind of variable, and not a flag-based variable (which I suspect it is)?